front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -95,8 +95,9 @@ export default function AssistantNew({
     isAgentConfigurationsLoading: isInitialAgentConfigurationsLoading,
   } = useAgentConfigurations({
     workspaceId: owner.sId,
-    agentsGetView: "workspace",
+    agentsGetView: "assistants-search",
     limit: 24,
+    includes: ["usage"],
```

Comment:
will this return the agents sorted by usage? since there's only 24 retrieved?
IMHO we should grab from redis the top 24 ids of most popular and fetch only those here (we can pass a list)
Happy to discuss IRL

Reply:
You right, just changed such that: the sorting is done when retrieving the ids from Redis